backend/mr-eventing/src/test/groovy/io/featurehub/mr/events/common/FeatureSetup.groovy

```diff
@@ -0,0 +1,57 @@
+package io.featurehub.mr.events.common
+
+import io.featurehub.db.model.DbApplication
+import io.featurehub.db.model.DbApplicationFeature
+import io.featurehub.db.model.DbEnvironment
+import io.featurehub.db.model.DbFeatureValue
+import io.featurehub.db.model.DbOrganization
+import io.featurehub.db.model.DbPerson
+import io.featurehub.db.model.DbPortfolio
+import io.featurehub.mr.model.FeatureValueType
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+ class FeatureSetup extends  Specification{
```

Comment:
why is this a Specification?

Reply:
I actually had to mock DbFeatureValue in the setup method because I needed to set whenUpdated which does not have a setter. Thats why this class has to extend Specification

Reply:
A specification has tests in it, this class has no tests in it, so is there any reason it can't just be a class?

Reply:
It looks like it can just be a method (as long as it returns a specific value) on FeatureMessagingCloudEventPublisherImplSpec. If it returns "def" then it is considered a test, if it returns a specific value, it isn't and is just a normal method.

Reply:
I wanted to reuse the setup between specs so I have now made this class into a base specification so the feature setup can be done from both the specs.

Reply:
lol - i think we are crossing wires here, I still don't get why its a specification at all, its just a method on a class 🤪 Lets get this merged :-)